ResourceManager/GitExtensionsControl.cs

```diff
@@ -123,7 +123,7 @@ public static bool IsTextEditKey(Keys keys, bool multiLine = false)
             keys &= ~Keys.Shift; // ignore the SHIFT key as modifier
             switch (keys)
             {
-                case Keys key when (key >= Keys.A && key <= Keys.Z) || (key >= Keys.D0 && key <= Keys.D9) || (key >= Keys.Oem1 && key <= Keys.Oem102):
+                case Keys key when key is >= Keys.A and <= Keys.Z or >= Keys.D0 and <= Keys.D9 or >= Keys.Oem1 and <= Keys.Oem102:
```

Comment:
I had to go and look for [docs](https://docs.microsoft.com/en-us/dotnet/csharp/language-reference/proposals/csharp-9.0/patterns3#pattern-combinators). It is legal, but IMO this is very error prone. `and` has a higher priority than `or`, but it is very subtle, and can lead to hard to find bugs. Please retain parens to enhance the readability

Reply:
Unfortunately StyleCop complains with parenthesis here...
For this specific lin

```
#pragma warning disable SA1008 // Opening parenthesis should be spaced correctly
                case Keys key when key is (>= Keys.A and <= Keys.Z) or (>= Keys.D0 and <= Keys.D9) or (>= Keys.Oem1 and <= Keys.Oem102):
#pragma warning restore SA1008 // Opening parenthesis should be spaced correctly
```

But I prefer separate lines:
```
                case Keys key when key is >= Keys.A and <= Keys.Z
                    or >= Keys.D0 and <= Keys.D9
                    or >= Keys.Oem1 and <= Keys.Oem102:
```